artifacts/centosstream/centos-stream_test.go

```diff
@@ -16,7 +16,7 @@ var _ = Describe("CentosStream", func() {
 	DescribeTable("Inspect should be able to parse checksum files",
 		func(release, mockFile string, details *api.ArtifactDetails,
 			exampleUserData *docs.UserData, envVariables map[string]string, metadata *api.Metadata) {
-			c := New(release, exampleUserData, envVariables)
+			c := New(release, "x86_64", exampleUserData, envVariables)
```

Comment:
IMO, I don't think it's needed. If it works with `x86_64`, it will work with `aarch64` and other CPU architectures. There is no different functionality if you enter `aarch64`.

Reply:
That isn't correct, there are changes to `Inspect` that produce different `ArtifactDetails` depending on the arch.